zerver/lib/bugdown/__init__.py

```diff
@@ -978,6 +978,11 @@ def extendMarkdown(self, md, md_globals):
             markdown.inlinepatterns.SimpleTagPattern(r'(\*\*)([^\n]+?)\2', 'strong'),
             '>not_strong')
 
+        md.inlinePatterns.add(
+            'italic',
+            markdown.inlinepatterns.SimpleTagPattern(r'(\*)([\w|\ |\t]+)\*', 'i'),
```

Comment:
Can you add a comment explaining the intended behavior and why this regular expression is correct?